algoliasearch/algoliasearch.go

```diff
@@ -172,6 +172,55 @@ type Client interface {
 	// also accepts extra RequestOptions.
 	MultipleQueriesWithRequestOptions(queries []IndexedQuery, strategy string, opts *RequestOptions) (res []MultipleQueryRes, err error)
 
+	// List all the cluster available for the application
+	ListClusters() (res []Cluster, err error)
+
+	// ListClustersWithRequestOptions is the same as ListClusters but it
+	// also accepts extra RequestOptions.
+	ListClustersWithRequestOptions(opts *RequestOptions) (res []Cluster, err error)
+
+	// List all user IDs across all clusters
+	ListUserIDs(page int, hitsPerPage int) (res listUserIDsRes, err error)
+
+	// ListUserIDsWithRequestOptions is the same as ListUserIDs but it
+	// also accepts extra RequestOptions.
+	ListUserIDsWithRequestOptions(page int, hitsPerPage int, opts *RequestOptions) (res listUserIDsRes, err error)
+
+	// Get a specific user ID
```

Comment:
Missing dot at the end.

I also find the comment a bit poor in term of meaning (compared to the name of the method ;p), perhaps we should say somewhere it's related to MCM for instance.